build.sbt

```diff
@@ -42,6 +50,11 @@ lazy val backend = (project in file("backend"))
             "-Xms128m", "-Xmx1024m"),
         // this enables custom javaOptions
         fork in run := true,
+        mappings in Universal ++= directory(baseDirectory.value / "sigar"),
+        bashScriptExtraDefines ++= Seq(
+          """declare -r sigar_dir="$(realpath "${app_home}/../sigar")"""",
+          """addJava "-Djava.library.path=${sigar_dir}""""
+        ),
```

Comment:
With `1.0.0` you can specify `jvmOpts` directly with `jvmOpts in Universal`. However with this the access to the installation path will get trickier, so this solution is pretty nice.